### PR TITLE
Lecture 9: Modeling Networks

### PR DESCRIPTION
**Pull request by HighlyAuditory**
Dear Xun,
We've finished our notes, please review. Thanks.

Rendered vesion
[https://llamasi.github.io/pgm-spring-2019/notes/lecture-09/](https://llamasi.github.io/pgm-spring-2019/notes/lecture-09/)

### CODE REVIEW

File: _posts/2019-02-13-lecture-09.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+  $$\lambda_s > C\sqrt{\frac{logp}{S}}$$
+
+  then with high possibility,
+
+  $$S(\hat{\beta}) \to S(\beta_{\ast})$$
+
+
+  [1]: https://projecteuclid.org/euclid.aos/1232115934
+
+### Why this algorithm work?
```
Comment (xunzheng) [inline _posts/2019-02-13-lecture-09.md]:
```suggestion
### Why does this algorithm work?
```

File: _posts/2019-02-13-lecture-09.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
+  After knowing the process of this algorithm to obtain the precision matrix through repeatedly applying LASSO regression on each node, we want to know why we can get the precision matrix in this way. In other words, we want to know why LASSO regression can select the neighborhood of each node. 
+
+#### Multivariate Gaussian
+  
+  For the multivatiate Gaussian there are several formulas to remember
+
+  $$p(x_2) = \mathcal{N}(x_2 | \mu_2, \Sigma_{22}) $$
+
+  $$p(x_1|x_2) = \mathcal{N}(x_2 | \mu_{1|2}, V_{1|2}) $$
```
Comment (xunzheng) [inline _posts/2019-02-13-lecture-09.md]:
```suggestion
  $$p(x_1|x_2) = \mathcal{N}(x_1 | \mu_{1|2}, V_{1|2}) $$
```

**Review by xunzheng — CHANGES_REQUESTED** [review]

**Comment by HighlyAuditory** [comment]
Can we merge now? Sorry but I'm not familiar with this process.

**Comment by alshedivat** [comment]
Some additional comments:
- Editor's name.
- Title of the notes is duplicated.
- Everything before section "GGM with Lasso" seems to be just a list of bullet points. Can you convert that to proper notes?
- References must be added using `<d-cite>` instead of links (see the template for examples).

**Comment by HighlyAuditory** [comment]
> Some additional comments:
> 
> * Editor's name.
> * Title of the notes is duplicated.
> * Everything before section "GGM with Lasso" seems to be just a list of bullet points. Can you convert that to proper notes?
> * References must be added using `<d-cite>` instead of links (see the template for examples).

Thanks for the advice. Please see the committed modification.

**Review by xunzheng — APPROVED** [review]

**Pull request merged**